lib/parser.js

```diff
@@ -783,19 +783,27 @@ export default function Parser(options) {
           continue;
         }
 
-        j = xml.indexOf('>', i + 1);
-        if (j === -1) {
-          return handleError('unclosed tag');
-        }
+        for (x = i + 1; ; x++) {
+          const tmpw = xml.charCodeAt(x);
+          if (isNaN(tmpw)) {
+            j = -1;
+            return handleError('unclosed tag');
+          }
 
-        // search for the last '>' before the '<' or the document end
-        // to close this tag. this allows us to safely parse '>' in
-        // attribute names, e.g. <foo name="bar>1" />
-        x = xml.indexOf('<', j + 1);
-        if (x === -1) {
-          j = xml.lastIndexOf('>');
-        } else {
-          j = xml.lastIndexOf('>', x);
+          // [10] AttValue ::= '"' ([^<&"] | Reference)* '"' | "'" ([^<&'] | Reference)* "'"
+          // skips the quoted string
+          // (double quotes) does not appear in a literal enclosed by (double quotes)
+          // (single quote) does not appear in a literal enclosed by (single quote)
+          if (tmpw === 34) { //  '"'
+            const tmpi = xml.indexOf('"', x + 1);
```

Comment:
Please drop `tmpi` and use existing temporary variable (or declare on top of scope).

Reply:
Thank you for pointing this out.
I declared the variable k at the top.
Then I changed it to use it to do the processing.